spiffe-helper.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: spiffe-helper
+  version: 0.9.0
+  epoch: 0
+  description: A helper utility for SPIFFE (Secure Production Identity Framework For Everyone) operations.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/spiffe/spiffe-helper.git
```

Comment:
we need to add commit here to make the CI happy because that is a required field.

Reply:
Done